proto/penumbra/buf.yaml

```diff
@@ -0,0 +1,18 @@
+version: v1
+name: buf.build/penumbra-zone/penumbra
+# We pin versions of the upstream cosmos deps, to avoid the chore of bumping
+# the lockfile regularly, while still detecting uncommitted changes to our
+# protos in CI.
+deps:
+  # https://buf.build/cosmos/ibc/docs/7ab44ae956a0488ea04e04511efa5f70, 2023-09-13
+  - buf.build/cosmos/ibc:7ab44ae956a0488ea04e04511efa5f70
+  # cosmos-sdk 9d547dbea90f47afbe1898388fcebffb, 2023-05-31, breaks the build, requiring
+  # attributes 'extension' and 'extension_signature' on 'proto::tendermint::types::Vote'.
+  # https://buf.build/cosmos/cosmos-sdk/docs/e7a85cef453e4b999ad9aff8714ae05f, 2023-05-22
+  - buf.build/cosmos/cosmos-sdk:e7a85cef453e4b999ad9aff8714ae05f
```

Comment:
My understanding is that these buf files were copied from the penumbra monorepo, but aren't actually being used to generate the protos. That's fine: the protos were copied too, and that means they definitely won't change on us in surprising ways. Eventually I'd like to use a fetch-from-BSR workflow, in line with our other repos, but we definitely don't need to do that today.